app/index.html

```diff
@@ -53,6 +53,7 @@
 
         <!-- build:js js/app.min.js -->
         <script src="components/socket.io-client/socket.io.js"></script>
+        <script src="components/jquery/dist/jquery.js"></script>
```

Comment:
Yeh I agree, I really didn't want to have to include it either. I'll look into alternative solutions.